nmostesting/IS04Utils.py

```diff
@@ -12,10 +12,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from . import TestHelper
 
 from .NMOSUtils import NMOSUtils
 
 
 class IS04Utils(NMOSUtils):
     def __init__(self, url):
         NMOSUtils.__init__(self, url)
+
+    def get_node(self):
```

Comment:
`IS04Utils` is also used with IS-04 Registration API and Query API.

Therefore, I think these functions need to be carefully named. Maybe the simplest change is just to change `get_node` to `get_self` which makes it clearer it works with the Node API. The other functions would work with the Query API at least.

Reply:
Ok understood. That makes sense.